test/views/InvalidTraderEoriViewSpec.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import models.{DraftId, NormalMode}
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.InvalidTraderEoriView
+
+class InvalidTraderEoriViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "invalidTraderEori"
+
+  val view: InvalidTraderEoriView = app.injector.instanceOf[InvalidTraderEoriView]
+
+  val viewViaApply: HtmlFormat.Appendable  = view(NormalMode, DraftId(1L), EoriNumber)(fakeRequest, messages)
```

Comment:
Since  EoriNumber used is from SpecBase, I'm guessing `draftId` was also added in SpecBase for reuse so can be used for all tests.

Reply:
Was going to make a constant file for these as noticed a lot of repetition for these in other tests too but maybe not in the PR.

Reply:
everything in the SpecBase would be moved to the constants file if needed. So I feel it should be done in this PR for easy transition and we can worry about the constant file later.